### ISSUE TITLE
Possible to add EFI 32 bit?

### ISSUE THREAD

**Issue by rudiservo** [thias/glim#79]
Is it possible to add 32 bit EFI boot?

**Comment by thias** [comment]
This seems very "niche". You can trivially do it by editing the `glim.sh` script if this is something you need.

Out of curiosity: Is this something you have really come across? Some 32bit computer being able to boot only using EFI and not legacy BIOS?

**Comment by rudiservo** [comment]
Yes, there are a lot of "windows" 32bit tablets from Acer, Asus, Lenovo, etc
Microsoft has come out and said they would discontinue support for them since a lot of them don't even have TPM2, I think that some Microsoft Surface models only boot EFI 32bit and windows 11 upgrade is out of the question.
IMO extending these devices usefulness will be by changing their OS, although they are mostly low power Atom cpus with 2 to 4 Gb ram, they have still some life in them.
